src/frontend/src/features/neighborhoods/neighborhoodsThunk.ts

```diff
@@ -0,0 +1,21 @@
+import { createAsyncThunk } from "@reduxjs/toolkit";
+import {
+    fetchNeighborhoods,
+    fetchNeighborhoodBounds,
+} from "../../api/neighborhoods";
+
+export const getNeighborhoodsAndBounds = createAsyncThunk(
+    "neighborhoods/getNeighborhoodsAndBounds",
+    async (authToken: string | null) => {
```

Comment:
I wonder if passing in `authToken` is a temporary solution or a pattern we'd like to keep. I am assuming the former, but would like to double check

Reply:
The former! This is also how the old codebase was setup but once we setup login/auth on the frontend I think we should access the token from the store directly when needed vs passing it down.